src/libponyc/pass/names.c

```diff
@@ -178,6 +178,15 @@ static bool names_typeparam(pass_opt_t* opt, ast_t** astp, ast_t* def)
     return false;
   }
 
+  if (ast_id(ast) == TK_NOMINAL) {
+    ast_t* module = ast_nearest(ast, TK_MODULE);
+    if (module && ast_get(module, ast_name(id), 0)) {
```

Comment:
Can you fix the style/formatting of both of these `if` statements to match the surrounding code style?

Specifically, no space between the `if` and the `(`, and the `{` should be on the next line.

It's not my favorite style for `if` statements, but it's the style we've got :wink:.